### PR TITLE
OE.md: add instructions for doing "32-bit userland/64-bit kernel" builds

### PR DESCRIPTION
**Pull request by andrey-konovalov**
This is a follow-up to https://github.com/96boards/oe-rpb-manifest/pull/54

I've put everything into OE.md, but the "### Creating the image for [Dragonboard-410c|Hikey]" parts could be split out if needed.

### CODE REVIEW

File: Reference-Platform/CECommon/OE.md [hunk 1]
```diff
@@ -226,6 +226,118 @@ If you boot this image on the board, you should get a command prompt on the HDMI
...
+`bitbake_secondary_image` actually runs two builds. So in the build directory,
+under `tmp-*/deploy/images/` two directories are created: one for 32-bit build
+artifacts, and the other for the 64-bit ones. E.g.
+```
+tmp-rpb_wayland-glibc/deploy/images/dragonboard-410c-32
+```
+and
+```
+tmp-rpb_wayland-glibc/deploy/images/dragonboard-410c
+
```
Comment (vchong) [inline Reference-Platform/CECommon/OE.md:282]: Missing ``` here?

**Comment by vchong** [comment]
Is this a pure 32-bit userland only or will it support a mixture of both 32- and 64-bit executables? If only 32, is there a way to support both?

**Comment by fixxxxxxer** [comment]
Thanks

File: Reference-Platform/CECommon/OE.md (hunk 1 above)
Comment (andrey-konovalov) [reply Reference-Platform/CECommon/OE.md:282]: yup... Let me fix that...

**Comment by andrey-konovalov** [comment]
On 11/01/2016 06:16 PM, vchong wrote:

> Is this a pure 32-bit userland only or will it support a mixture of both 32- and 64-bit executables? If only 32, is there a way to support both?

Pure 32-bit userland only.
There is no support for creating a mix of of both 32- and 64-bit executables.

**Pull request merged**